lnbits/wallets/zbd.py

```diff
@@ -64,18 +65,25 @@ async def create_invoice(
         **kwargs,
     ) -> InvoiceResponse:
         # https://api.zebedee.io/v0/charges
-        if description_hash or unhashed_description:
-            raise UnsupportedError("description_hash")
-
+        
         msats_amount = amount * 1000
         data: Dict = {
             "amount": f"{msats_amount}",
-            "description": memo,
+            "description": "",
             "expiresIn": 3600,
             "callbackUrl": "",
             "internalId": "",
         }
 
+        ## handle description_hash and unhashed for ZBD
+        if description_hash:
+            data["description"] = description_hash.hex()
+        elif unhashed_description:
+            data["description"] = hashlib.sha256(unhashed_description).hexdigest()
```

Comment:
other sources use `descriptionHash` for the hashes - I think we should do the same here?

```suggestion
            data["descriptionHash"] = description_hash.hex()
        elif unhashed_description:
            data["descriptionHash"] = hashlib.sha256(unhashed_description).hexdigest()
```

Reply:
the ZBD docs ask for "description" , not descriptionHash

see here: https://zbd.dev/api-reference/zbd-gamertag/create

Reply:
i see a ```    "invoiceDescriptionHash": null ``` but on the response, the docs are weird, i think its fine to only send description to zbd

Reply:
OK